liana/src/spend.rs

```diff
@@ -55,6 +55,8 @@ pub enum SpendCreationError {
     SanityCheckFailure(Psbt),
     FetchingTransaction(bitcoin::OutPoint),
     CoinSelection(InsufficientFunds),
+    // TODO: wrap more specific error
```

Comment:
Is there actually a more specific error we can wrap? This feels like a item that came from the hackathon